src/main/java/com/thealgorithms/conversions/UnitConversions.java

```diff
@@ -0,0 +1,14 @@
+package com.thealgorithms.conversions;
+
+import static java.util.Map.entry;
+
+import java.util.Map;
+import org.apache.commons.lang3.tuple.Pair;
+
+public final class UnitConversions {
+    private UnitConversions() {
+    }
+
+    public static final UnitsConverter TEMPERATURE = new UnitsConverter(Map.ofEntries(entry(Pair.of("Kelvin", "Celsius"), new AffineConverter(1.0, -273.15)), entry(Pair.of("Celsius", "Fahrenheit"), new AffineConverter(9.0 / 5.0, 32.0)),
```

Comment:
As you mentioned "is it fine to represent units as Strings?". 
What do you think about introduce enum here:

`public enum TemperatureUnit {
    KELVIN,
    CELSIUS,
    FAHRENHEIT,
    REAUMUR,
    DELISLE,
    RANKINE
}
` 
If you want to save correctness for naming have enums with specialized language description for each.
Something like this:

```
public enum TemperatureUnit {
    KELVIN("Kelvin");
    private final String unit;
    
    TemperatureUnit(String unit) {
        this.unit = unit;
    }
    
    public String getUnit() {
        return unit;
    }
}
```

Enums would provide type safety, enhance readability, and make it easier to manage different unit systems. And the listings should not overly complicated this framework.

Reply:
This will be cumbersome: think about other units like meters, inches, etc.